source/WindowScroller/WindowScroller.js

```diff
@@ -127,6 +131,8 @@ export default class WindowScroller extends PureComponent {
 
   // Referenced by utils/onScroll
   __handleWindowScrollEvent (event) {
+    if (!this._isMounted) return
```

Comment:
@liorbrauer I think ideally we should determine why the mountedInstances array is not accurate, rather than fixing it with a flag

https://github.com/bvaughn/react-virtualized/blob/master/source/WindowScroller/utils/onScroll.js